modules/overrides/list_to_map.go

```diff
@@ -15,7 +15,7 @@ var _ json.Unmarshaler = (*ListToMap)(nil)
 
 // MarshalYAML implements the Marshal interface of the yaml pkg.
 func (l ListToMap) MarshalYAML() (interface{}, error) {
-	list := make([]string, 0)
+	list := make([]string, 0, len(l))
 	for k := range l {
 		list = append(list, k)
```

Comment:
I think if we have the slice of size len(l), then we'll also want to use the index to set the list values rather than append.  Append will continue to grow the size of the slice by starting at the end.

Reply:
this change sets the capacity, not the length. it will be fine.